UserAgent/Views/Privacy Indicator/PrivacyDashboardView.swift

```diff
@@ -2,27 +2,47 @@ import UIKit
 import Shared
 
 class PrivacyDashboardView: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.snp.makeConstraints { make in
+            // This fixes a bug
+            // where the tableview would squash elements inside PrivacyDashboardView
+            make.height.greaterThanOrEqualTo(UIScreen.main.bounds.height * 0.3)
```

Comment:
:+1: height should be a passed as constructor argument or maybe inferred from frame

Reply:
@chrmod @pavel-cliqz  guys, I am a bit confused what should we do: 
1) pass height as argument -- agree, can it be done in the next pr (because it does not come from me :)      
2) inferred from frame -- let look how Client use:  ```let trackerInfoView = PrivacyDashboardView()``` I think ```frame``` in this case some empty object (0.0, 0.0, 0.0, 0.0)
3) dont change, but check rotation --- seems fine (dashboard height is bigger then display height -- but there is scroll)